test/mock-cloud.py

```diff
@@ -24,39 +24,39 @@
 import argparse
 import secrets
 
-cloud_exchange = 'connIn'
-fog_exchange = 'connOut'
+#AMQP Configuration values:
+data_exchange = 'data.sent'
+device_exchange = 'device'
 
 QUEUE_CLOUD_NAME = 'connIn-messages'
 
 MESSAGE_EXPIRATION_TIME_MS = '10000'
-
+# Babeltower Events API - v2.0.0
```

Comment:
Nit: I like to avoid adding version numbers in code because they tend to get outdated somewhat quickly.
Maybe, because this is an API, it might be a fine, but I'd still at least remove the minor and patch version numbers, since these changes won't break compatibility.